packages/fxa-graphql-api/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "fxa-graphql-api",
+  "version": "1.154.0",
+  "description": "FxA GraphQL API",
+  "scripts": {
+    "build": "./node_modules/typescript/bin/tsc",
+    "lint": "npm-run-all --parallel lint:*",
+    "audit": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-",
+    "lint:tslint": "./node_modules/tslint/bin/tslint -p .",
+    "watch": "tsc -w",
+    "start": "pm2 start pm2.config.js",
+    "stop": "pm2 stop pm2.config.js",
+    "restart": "pm2 restart pm2.config.js",
+    "test": "./node_modules/mocha/bin/mocha -r ts-node/register src/test/**/*.spec.ts src/test/**/**/*.spec.ts src/test/**/**/**/*.spec.ts",
+    "email-bounce": "ts-node ./scripts/email-bounce.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/fxa.git"
+  },
+  "engines": {
+    "node": ">=12"
+  },
+  "author": "Mozilla (https://mozilla.org/)",
+  "license": "MPL-2.0",
+  "bugs": {
+    "url": "https://github.com/mozilla/fxa/issues"
+  },
+  "homepage": "https://github.com/mozilla/fxa#readme",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "@sentry/integrations": "^5.15.5",
+    "@sentry/node": "^5.15.5",
+    "apollo-server": "^2.12.0",
+    "apollo-server-express": "^2.12.0",
+    "convict": "^5.2.0",
+    "graphql": "^14.6.0",
+    "knex": "^0.21.0",
+    "mozlog": "^2.2.0",
+    "mysql": "^2.18.1",
+    "objection": "^2.1.3",
+    "reflect-metadata": "^0.1.13",
+    "tslib": "^1.11.1",
+    "type-graphql": "^0.17.6",
+    "typedi": "^0.8.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.11",
+    "@types/chance": "^1.0.10",
+    "@types/convict": "^4.2.1",
+    "@types/graphql": "^14.5.0",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^13.13.2",
```

Comment:
afaik we are using node 12 locally for development and in docker images, so preferably we should use the 12.x types here.